deploy/examples/operator.yaml

```diff
@@ -461,6 +461,9 @@ spec:
             - name: ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS
               value: "5"
 
+            - name: ROOK_DISABLE_ADMISSION_CONTROLLER
```

Comment:
We will also need a setting for this in the helm chart

Reply:
added